models/User.js

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
We said we won't have a user schema.
We will use auth0's api to manage our users.